src/app/screens/section-input/section-input.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit, Input, forwardRef } from '@angular/core';
+import { NG_VALUE_ACCESSOR, ControlValueAccessor, FormControl, NG_VALIDATORS } from '@angular/forms';
+
+@Component({
+  selector: 'app-section-input',
+  templateUrl: './section-input.component.html',
+  styleUrls: ['./section-input.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => SectionInputComponent),
+      multi: true
+    },
+    {
+      provide: NG_VALIDATORS,
+      useExisting: forwardRef(() => SectionInputComponent),
+      multi: true,
+    }
+  ]
+})
+export class SectionInputComponent implements OnInit, ControlValueAccessor {
+
+  constructor() { }
+
+  @Input() myLabel;
+  @Input() typeInput;
+  @Input() myTextValue = ''
+  fcontrol: FormControl;
+  messageAlert: string = 'Campo obligatorio'
+
+  onChange = (_: any) => {}
+  onTouch = () => { }
+
+  ngOnInit() {}
+
+  onBlur(){
+    this.onTouch()
```

Comment:
Why are you calling this event for? It seems that the function onTouch doing nothing